<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocaleModule</class>
 <widget class="QWidget" name="LocaleModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAdd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRemove">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRestore">
       <property name="text">
        <string>Restore</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="languageTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="localeBasicTab">
      <attribute name="title">
       <string>System Locales</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QScrollArea" name="localeBasicScrollArea">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="localeBasicScrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>560</width>
            <height>338</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QListView" name="localeListView">
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="localeAdvancedTab">
      <attribute name="title">
       <string>Detailed Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QScrollArea" name="localeAdvancedScrollArea">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="localeAdvancedScrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>546</width>
            <height>500</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QGroupBox" name="displayLanguageGroupBox">
             <property name="title">
              <string>Display Language</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QFormLayout" name="formLayout_2">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
              </property>
              <property name="labelAlignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="langLabel">
                <property name="text">
                 <string>Language:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="langComboBox"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="collateLabel">
                <property name="text">
                 <string>Collation and Sorting:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="collateComboBox"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="messagesLabel">
                <property name="text">
                 <string>Messages:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="messagesComboBox"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="ctypeLabel">
                <property name="text">
                 <string>CType:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QComboBox" name="ctypeComboBox"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="formatsGroupBox">
             <property name="title">
              <string>Formats</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
              </property>
              <property name="labelAlignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="numericLabel">
                <property name="text">
                 <string>Numbers:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="numericComboBox"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="timeLabel">
                <property name="text">
                 <string>Time:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="timeComboBox"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="monetaryLabel">
                <property name="text">
                 <string>Currency:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="monetaryComboBox"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="measurementLabel">
                <property name="text">
                 <string>Measurement Units:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QComboBox" name="measurementComboBox"/>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="addressLabel">
                <property name="text">
                 <string>Address:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QComboBox" name="addressComboBox"/>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="nameLabel">
                <property name="text">
                 <string>Names:</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QComboBox" name="nameComboBox"/>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="telephoneLabel">
                <property name="text">
                 <string>Telephone:</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QComboBox" name="telephoneComboBox"/>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="identificationLabel">
                <property name="text">
                 <string>Identification:</string>
                </property>
               </widget>
              </item>
              <item row="7" column="1">
               <widget class="QComboBox" name="identificationComboBox"/>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="paperLabel">
                <property name="text">
                 <string>Paper:</string>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="QComboBox" name="paperComboBox"/>
              </item>
              <item row="9" column="1">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="setRegionAndFormatsAction">
   <property name="text">
    <string>Set as default display language and format</string>
   </property>
  </action>
  <action name="setRegionAction">
   <property name="text">
    <string>Set as default display language</string>
   </property>
  </action>
  <action name="setFormatsAction">
   <property name="text">
    <string>Set as default format</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
